src/ViewInstance.js

```diff
@@ -641,7 +570,7 @@ class ViewInstance extends React.Component {
       acc03: !areAllFieldsEmpty([identifiers]),
       acc04: !areAllFieldsEmpty([contributors]),
       acc05: !areAllFieldsEmpty(values(descriptiveData)),
-      acc06: !areAllFieldsEmpty([instanceNotes]),
+      acc06: !areAllFieldsEmpty([[get(instance, 'notes', [])]]),
```

Comment:
@NikitaSedyx could we use `instance?.nodes ?? []` instead?

Reply:
yep, will change